README.md

```diff
@@ -1,160 +1,718 @@
+2. **Elastic Site Search**: [search-ui-site-search-connector](packages/search-ui-site-search-connector)
 
-Dependencies are declared in a package.json hierarchy.
+The [example search UI](#search-ui) above uses the Elastic App Search Connector:
 
-- / package.json - Dependencies for repo tooling, like `husky` and `lerna`.
-- /packages package.json - Common dev dependencies for all Search UI npm packages. Any dev Common dev dependencies for al that does not need to be called directly in a package level npm command
-  can be declared here.
-- /packages/{package_name} package.json - Package specific dependencies.
+```js
+const connector = new AppSearchAPIConnector({
+  searchKey: "search-371auk61r2bwqtdzocdgutmg",
+  engineName: "search-ui-examples",
+  hostIdentifier: "host-2376rb"
+});
+```
 
-Note that we do not encourage "hoisting" dependencies through lerna. This WILL
-cause the examples applications to error out from dependency version conflicts.
+Search UI can connect to **any** web based Search API. Read [the wiki](https://github.com/elastic/search-ui/wiki/Build-Your-Own-Connector) for more information.
 
-<a id="install"></a>
+### 2. SearchProvider
 
-### Installing dependencies
+The `SearchProvider` object will tie all of your Components together so that they
+work as a cohesive application. As users take actions using your Components,
+like submitting a search box or applying a filter, `SearchProvider` will then
+trigger API calls via the Connector you've configured to fetch Search Results.
 
-From the root level of this repository, run the following commands in order:
+The flow is like this:
 
-```shell
-# Install top level depenedencies at the root of the project
-npm install
+```
+Components -> SearchProvider -> Connector -> Search API (App Search)
+```
 
-# Install dependencies in the /packages directory
-(cd packages && npm install)
+`SearchProvider` is lightweight:
 
-# Install all dependencies for the individual packages
-npm run bootstrap
+```jsx
+<SearchProvider
+  config={{
+    apiConnector: connector
+  }}
+>
+  {() => <div className="App">{/* Place Components here! */}</div>}
+</SearchProvider>
 ```
 
-### Building
+### 3. Components
+
+Components are the building blocks from which craft your search experience.
+
+Each Component - like `SearchBox` and `Results` - is a child of the `SearchProvider` object:
+
+```jsx
+<SearchProvider
+  config={{
+    apiConnector: connector
+  }}
+>
+  {() => (
+    <div className="App">
+      <Layout
+        header={<SearchBox />}
+        bodyContent={<Results titleField="title" urlField="nps_link" />}
+      />
+    </div>
+  )}
+</SearchProvider>
+```
+
+The following Components are available:
+
+- [SearchBox](#searchbox)
+- [Results](#results)
+- [ResultsPerPage](#resultsperpage)
+- [Facet](#facet)
+- [Sorting](#sorting)
+- [Paging](#paging)
+- [PagingInfo](#paginginfo)
+- [ErrorBoundary](#errorboundary)
+
+---
+
+#### SearchBox
```

Comment:
If we're focusing on keep this README short and concise, I think we could move the Component Reference elsewhere.